src/Apps/Order/Routes/NewPayment/index.tsx

```diff
@@ -89,6 +106,25 @@ export class NewPaymentRoute extends Component<
       if (orderOrError.error) {
         this.handleFixFailedPaymentError(orderOrError.error.code)
         return
+      } else if (
+        orderOrError.actionData &&
+        orderOrError.actionData.clientSecret
+      ) {
+        this.state.stripe
+          .handleCardAction(orderOrError.actionData.clientSecret)
+          .then(scaResult => {
```

Comment:
this promise chain should probably be changed to use `await` to be consistent with above and also to make use of the general error handling in this method.